data/net/minecraft/world/inventory/MerchantMenu.mapping

```diff
@@ -1,22 +1,39 @@
 CLASS net/minecraft/world/inventory/MerchantMenu
+	METHOD <init> (ILnet/minecraft/world/entity/player/Inventory;)V
+		ARG 1 containerId
+		ARG 2 playerInventory
+	METHOD <init> (ILnet/minecraft/world/entity/player/Inventory;Lnet/minecraft/world/item/trading/Merchant;)V
+		ARG 1 containerId
+		ARG 2 playerInventory
+		ARG 3 merchant
```

Comment:
Point of 'parameters follow field names': `merchant` -> `trader`.
(Though I must admit that this would be inconsistent a bit with `MerchantContainer` and `merchant`, but alas.)